examples/src/main/java/tachyon/examples/BasicNonByteBufferOperations.java

```diff
@@ -135,20 +138,22 @@ private boolean read(TachyonFileSystem tachyonFileSystem, TachyonURI filePath,
   }
 
   public static void main(final String[] args) throws IOException {
-    if (args.length < 2 || args.length > 6) {
+    if (args.length < 2 || args.length > 7) {
       usage();
     }
 
     Utils.runExample(new BasicNonByteBufferOperations(new TachyonURI(args[0]),
         new TachyonURI(args[1]), Utils.option(args, 2, TachyonStorageType.STORE),
-        Utils.option(args, 3, TachyonStorageType.NO_STORE), Utils.option(args, 4, true),
-        Utils.option(args, 5, 20)));
+        Utils.option(args, 3, UnderStorageType.NO_PERSIST),
+        Utils.option(args, 4, TachyonStorageType.NO_STORE), Utils.option(args, 5, true),
+        Utils.option(args, 6, 20)));
   }
 
   private static void usage() {
     System.out.println("java -cp " + Constants.TACHYON_JAR + " "
         + BasicNonByteBufferOperations.class.getName() + " <master address> <file path> "
```

Comment:
`BasicNonByteBufferOperations.class.getName() + " <master address> <file path>  <tachyon storage type for writes (STORE | NO_STORE)> <under storage type for writes (SYNC_PERSIST | NO_PERSIST)> <tachyon storage type for reads (STORE | NO_STORE)> <delete file> <number of files>"`